spec.md

```diff
@@ -333,10 +333,15 @@ service Node {
     returns (NodeUnpublishVolumeResponse) {}
 
   rpc NodeGetId (NodeGetIdRequest)
-    returns (NodeGetIdResponse) {}
+    returns (NodeGetIdResponse) {
+    option deprecated = true;
```

Comment:
Please include a comment directing the reader to the new API. Also, this PR schedules this RPC for removal prior to publishing CSI v1.0... should include such comment

Reply:
We should make it clear that prior to v1.0 plugins that implement NodeGetInfo MUST implement this as well